tools/llm_bench/task/text_to_speech_generation.py

```diff
@@ -230,7 +231,8 @@ def run_text_2_speech_benchmark(model_path, framework, device, args, num_iters,
         for idx, input_text in enumerate(text_list):
             p_idx = prompt_idx_list[idx]
             for num in range(num_iters + 1):
-                if num == 0:
+                if args["output_dir"] is None and num == 0:
```

Comment:
please clarify - if we save generated speech to output_dir, right? If yes, I expect that input text will be showed despite output_dir is set. Now it is more convinient.